index.js

```diff
@@ -248,46 +249,89 @@ module.exports = {
       )
     }
 
-    // Listeners for joining groups
-    function start() {
-      ssb.metafeeds.findOrCreate((err, myRoot) => {
-        if (err) throw new Error('Could not find or create my root feed')
-        findOrCreateAdditionsFeed((err) => {
-          if (err) console.warn('Error finding or creating additions feed', err)
-        })
-
-        pull(
-          ssb.db.query(
-            where(and(isDecrypted('box2'), type('group/add-member'))),
-            live({ old: true }),
-            toPullStream()
-          ),
-          pull.filter((msg) =>
-            lodashGet(msg, 'value.content.recps', []).includes(myRoot.id)
-          ),
-          pull.asyncMap((msg, cb) => {
-            const groupId = lodashGet(msg, 'value.content.recps[0]')
-
-            ssb.box2.getGroupKeyInfo(groupId, (err, info) => {
-              if (err) {
-                console.error('Error when finding group invite for me:', err)
-                return
-              }
+    function listInvites() {
+      return pull(
+        pull.values([0]),
+        pull.asyncMap((n, cb) => {
+          ssb.metafeeds.findOrCreate((err, myRoot) => {
+            if (err) throw new Error('Could not find or create my root feed')
+
+            ssb.box2.listGroupIds((err, groupIds) => {
+              if (err) throw err
+
+              return cb(
+                null,
+                pull(
+                  ssb.db.query(
+                    where(and(isDecrypted('box2'), type('group/add-member'))),
+                    toPullStream()
+                  ),
+                  pull.filter((msg) =>
+                    // it's an addition of us
+                    lodashGet(msg, 'value.content.recps', []).includes(
+                      myRoot.id
+                    )
+                  ),
+                  pull.filter(
+                    (msg) =>
+                      // we haven't already accepted the addition
+                      !groupIds.includes(
+                        lodashGet(msg, 'value.content.recps[0]')
```

Comment:
Just a comment that doesn't require your action in this PR: it would be nice to get rid of lodashGet here and use optional chaining, supported in Node.js 14+ (we are targeting 16+):

`msg?.value?.content?.recps?.[0]`